src/test/java/core/basesyntax/service/impl/ReaderServiceImplTest.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReaderService;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class ReaderServiceImplTest {
+    private static final String VALID_TEST_FILE_PATH
+            = "src/test/resources/validDataTestFile.csv";
+    private ReaderService readerService;
+
+    @BeforeEach
+    void setUp() {
+        readerService = new ReaderServiceImpl();
```

Comment:
Do you need to create a new instance before each test method? Maybe it will be better to create **before all** test methods execution.